src/Validation/Validation.php

```diff
@@ -216,6 +216,36 @@ public static function cc($check, $type = 'fast', $deep = false, $regex = null)
     }
 
     /**
+     * Used to check the count of a given value of type string, int, or array.
+     *
+     * If a string value is passed the string length is used as count.
+     *
+     * @param array|int|string $check1 The value to check the count on.
+     * @param string $operator Can be either a word or operand
+     *    is greater >, is less <, greater or equal >=
+     *    less or equal <=, is less <, equal to ==, not equal !=
+     * @param int $expectedCount The expected count value.
+     * @return bool Success
+     */
+    public static function count($check1, $operator, $expectedCount)
+    {
+        $count = null;
+        if (is_array($check1) || $check1 instanceof \Countable) {
+            $count = count($check1);
+        } elseif (is_string($check1)) {
+            $count = mb_strlen($check1);
+        } elseif (is_int($check1)) {
```

Comment:
`Validation::equals()` validates numbers, `Validation::lengthBetween()` validates string length. This method should only validate stuff that can be safely iterated.

If you think we need more / better methods for strings or integers, you are very welcome to add them.